docs/dynamic-tao/dtao-guide.md

```diff
@@ -70,9 +70,9 @@ Each block:
 - the chain emits TAO and injects it into the TAO reserves of the subnets.
 <!-- (how much TAO???) -->
 
-- the chain emits 2 each alpha currency, distributed between:
+- the chain emits `2 ×` (2 times)  the alpha emission rate for each subnet. The alpha emission rate follows a halving schedule (exactly like Bitcoin), decreasing over time. The emitted alphas are distributed between:
```

Comment:
Just confirming that you mean *the emission schedule is the same as bitcoin*, i.e., initially 50 alpha per block in with a halving every 210000 blocks? That's how the sentence reads to me. Or just that it's similar in principle but with different numbers? Either way, let's give the numbers for the emission halving schedule if we can please.

Reply:
yes exactly, although i should've said exactly like TAO. when a subnet's alpha reached 10,500,000 in circulation, each block will emit 1 alpha in total instead of 2.